README.md

```diff
@@ -49,6 +49,24 @@ Download the latest release [https://github.com/andredumas/techan.js/releases/la
 npm install --save techan
 ```
 
+#### Webpack (ES6)
+```javascript
+// webpack.config.js
+config = {
+    ...,
+    resolve: {
+        ...,
+        alias: {
+            'techan': path.resolve('node_modules/techan/src/techan.js')
+        }
+    }
+}
+
+// usage
+import * as d3 from 'd3';
```

Comment:
I just removed my README modifications all-together. From my research, it seems like the problem i was experiencing is a bug in babel and unrelated to your library. The workaround was to simply include d3 with a typical script tag, outside the flow of babel.